.github/workflows/migration-pipeline.yml

```
@@ -0,0 +1,282 @@
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-setuptools python3-pip python3-dev build-essential python-is-python3
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Run unit tests
+        run: npm run test:unit -- --coverage
+
+      - name: Upload coverage reports
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)